src/rpc/mempool.cpp

```diff
@@ -384,6 +427,49 @@ UniValue MempoolToJSON(const CTxMemPool& pool, bool verbose, bool include_mempoo
     }
 }
 
+static RPCHelpMan getmempoolfeeratediagram()
+{
+    return RPCHelpMan{"getmempoolfeeratediagram",
+        "Returns the feerate diagram for the whole mempool.",
+        {},
+        {
+            RPCResult{"mempool chunks",
+                RPCResult::Type::ARR, "", "",
+                {
+                    {
+                        RPCResult::Type::OBJ, "", "",
+                        {
+                            {RPCResult::Type::NUM, "vsize", "cumulative vsize"},
+                            {RPCResult::Type::NUM, "fee", "cumulative fee"}
```

Comment:
nit, also happy to put in followup

```suggestion
                            {RPCResult::Type::NUM, "vsize", "cumulative sigops-adjusted vsize"},
                            {RPCResult::Type::NUM, "fee", "cumulative modified fee"}
```